### PR TITLE
ci: remove codecov install and execution

### PR DESCRIPTION
**Pull request by mojito317**
## PR summary

Currently we are not getting any benefit from the `codecov` integration and  due to [a recent security incident](https://about.codecov.io/security-update/?utm_medium=email&utm_source=marketo&ajs_uid=&ajs_event=Clicked%20Email&mkt_tok=MzMyLUxWWC03NDEAAAF8dJnGkNG_ojrjdXL2M-fOtB8t5IIphrIeLMth6mxIfuhPHB7b_FDof-GXtPcIc5fc754EGZ-_sxnLPdyBwLd1yNBoppIr0iZuG9A5_FAL) we decided to remove it.

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [x] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [x] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,7 +35,6 @@ before_install:
 - bash scripts/setup_couch.sh
 
 install:
```
Comment (ricellis) [inline .travis.yml:37]: Can we remove the whole `install` block as we don't appear to be installing anything else?
```suggestion
```

**Review by ricellis — APPROVED** [review]
+1, but remove the install block if it suits

File: .travis.yml (hunk 1 above)
Comment (mojito317) [reply .travis.yml]: Done in 0c62f16.

**Review by emlaver — APPROVED** [review]

**Comment by mojito317** [comment]
Thanks for the review @ricellis and @emlaver! I'm about to merge this!

**Pull request merged**